### PR TITLE
Initial repository structure

### PR DESCRIPTION
**Pull request by ptoscano**
Mostly copied from the 'template' repository, enough to be an empty role, and run the tox-lsr CI for it.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/articles/about-code-owners
+# Default reviewers for everything
+* @spetrosi @nhosoi @richm
```
Comment (richm) [inline .github/CODEOWNERS:3]: you can omit spetrosi and nhosoi - but add yourself to this list

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  min_ansible_version: "2.9"
+
+  platforms:
+    - name: Fedora
+      versions:
+        - all
+    - name: EL
+      versions:
+        - 9
```
Comment (richm) [inline meta/main.yml:18]: Should the rhc role support rhel 7 or 8?

File: .github/CODEOWNERS (hunk 1 above)
Comment (ptoscano) [reply .github/CODEOWNERS:3]: ack, will change

File: meta/main.yml (hunk 2 above)
Comment (ptoscano) [reply meta/main.yml:18]: at least rhel 8 yes, maybe rhel 7; i want to start small for now, and add them later on

**Review by richm — APPROVED** [review]

**Pull request merged**